From: Chad Monroe <chad@monroe.io>
Date: Sun, 26 Jul 2026 20:29:06 -0700
Subject: [PATCH] wifi: mt76: eeprom: log nvmem cell name and size on load

The EEPROM file loading path and its diagnostics were dropped in
favor of nvmem cells; this is the replacement breadcrumb showing
which cell (eeprom, precal) was consumed at boot and how many
bytes were taken.

Fixes: SDG-2651

Signed-off-by: Chad Monroe <chad@monroe.io>
---
 eeprom.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/eeprom.c
+++ b/eeprom.c
@@ -144,6 +144,8 @@ int mt76_get_of_data_from_nvmem(struct m
 
 	memcpy(eep, data, len);
 
+	dev_info(dev->dev, "nvmem cell: %s loaded %d bytes\n", cell_name, len);
+
 exit:
 	kfree(data);
 
